state-chain/pallets/cf-pools/src/tests.rs

```diff
@@ -241,17 +238,9 @@ fn test_buy_back_flip() {
 		// If we're at an interval, we should buy flip.
 		LiquidityPools::on_initialize(INTERVAL * 3);
 		assert_eq!(0, CollectedNetworkFee::<Test>::get());
-		assert_eq!(
-			MockSwapQueueApi::get_swap_queue()
-				.first()
-				.expect("Should have scheduled a swap usdc -> flip"),
-			&MockSwap {
-				from: STABLE_ASSET,
-				to: FLIP,
-				amount: EXPECTED_COLLECTED_FEES,
-				swap_type: SwapType::NetworkFee,
-			}
-		);
+
+		// TODO: check that a swap is scheduled once we network fee stuff
```

Comment:
```suggestion
		// TODO: check that a swap is scheduled once the network fee stuff
```

Reply:
Jamie's network fee PR has been moved, so this is no longer relevant :)